tests/Tutor.Web.Tests/Integration/Domain/FeedbackTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tutor.Core.DomainModel.Feedback;
+using Tutor.Infrastructure.Database;
+using Tutor.Web.Controllers.Domain;
+using Tutor.Web.Controllers.Domain.DTOs.Feedback;
+using Xunit;
+
+namespace Tutor.Web.Tests.Integration.Domain
+{
+    [Collection("Sequential")]
+    public class FeedbackTests : BaseIntegrationTest
+    {
+        public FeedbackTests(TutorApplicationTestFactory<Startup> factory) : base(factory) {}
+
+        [Theory]
+        [MemberData(nameof(EmotionsFeedbackSubmission))]
+        public void Stores_emotions_feedback(EmotionsFeedbackDto postedFeedback, EmotionsFeedbackDto expectedFeedback)
```

Comment:
I assume that all tests in the projects are passing?

Reply:
Added test and all tests from the `Domain` sub-folder are passing. 
Only test that's falling on my machine is `Can_transform_to_sql` from `Infrastructure` sub-folder, due to the way the test is written.